### PR TITLE
Suggest creating a new release after merging a PR

### PR DESCRIPTION
**Pull request by fregante**
- Closes https://github.com/refined-github/refined-github/issues/3619

## Test URLs

- Any merged PR that hasn't been released, like https://github.com/refined-github/refined-github/pull/5767
- Any PR you're about to merge

## Screenshot

https://user-images.githubusercontent.com/1402241/177039698-0e454ff4-cb53-407c-9104-b760aee9607d.mov

### CODE REVIEW

File: source/features/first-published-tag-for-merged-pr.tsx [hunk 1]
```diff
@@ -24,10 +31,14 @@ async function init(): Promise<void> {
 	const mergeCommit = select(`.TimelineItem.js-details-container.Details a[href^="/${getRepo()!.nameWithOwner}/commit/" i] > code`)!.textContent!;
 	const tagName = await getFirstTag(mergeCommit);
 
-	if (!tagName) {
-		return;
+	if (tagName) {
+		addExistingTagLink(tagName);
+	} else if (canCreateRelease()) {
+		addLinkToCreateRelease('This PR doesn’t appear to have been released yet');
```
Comment (fregante) [inline source/features/first-published-tag-for-merged-pr.tsx:37]: Screen Shot 12

File: source/github-events/on-pr-merge.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import select from 'select-dom';
+
+import observeElement from '../helpers/simplified-element-observer';
+
+export default function onPrMerge(callback: VoidFunction): Deinit {
```
Comment (fregante) [inline source/github-events/on-pr-merge.ts:5]: Can be used in `pr-branch-auto-delete` too

File: source/github-helpers/timeline-item.tsx [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import React from 'dom-chef';
+
+export default function TimelineItem(): JSX.Element {
```
Comment (fregante) [inline source/github-helpers/timeline-item.tsx:3]: Overlap with #5766

**Comment by fregante** [comment]
Merged but feel free to review and suggest copy changes, etc. I just wanted to unblock https://github.com/refined-github/refined-github/pull/5766

**Comment by fregante** [comment]
> really going in the way

It's a notice, you can ignore it. I don't see it as a big issue

**Comment by yakov116** [comment]
Come to think of it why is it part of one feature?

**Comment by fregante** [comment]
Because they do the same thing:

- has this PR been released?
	- yes: link to it
	- no: can the current user release it?
		- yes: link to new release
		- no: 🤫

**Comment by lode** [comment]
Not sure if this is the place to discuss, but I also find the suggest to create a release quite annoying. Mostly on a private repo which doesn't have any releases and won't get any in the future either. Is it possible to hide it when the repo doesn't use releases at all?

I also find it a little hard to hide via css, since the code example posted seems very fragile.

**Comment by yakov116** [comment]
Can you make a new issue?

**Comment by fregante** [comment]
I don’t get the point. There are a hundred buttons that you don’t use on the page, are they all tempting you? There’s a big “Close and comment” button in every conversation, but you rarely do that. Should it be hidden? 🤷‍♂️

**Comment by yakov116** [comment]
This is a banner ot sticks out

**Pull request merged**